### ISSUE TITLE
Are the weights available?

### ISSUE THREAD

**Issue by NightMachinery** [mlzxy/devit#2]

**Comment by mlzxy** [comment]
Yes. Please check https://github.com/mlzxy/devit/blob/main/Downloads.md.  It has pretrained weights, training log, and some annotated ycb images.

**Comment by mdfwn** [comment]
`Access denied with the following error: Cannot retrieve the public link of the file. You may need to change the permission to "Anyone with the link", or have had many accesses.`

**Comment by mlzxy** [comment]
Hi @mdfwn This is the screenshot of opening the given link from a non-logined firefox:

![image](https://github.com/mlzxy/devit/assets/7659710/1961eae6-15c1-4387-b558-9d52024ed693)

I am pretty sure at least some people could access the link publicly. Could you share a screenshot of yours opening the link?

**Comment by mlzxy** [comment]
@mdfwn  

If `gdown` is the issue, please use `rclone` following the instruction from #7. Let me know if you have any questions!
